Confus/Confus/Networking/ClientConnection.h

```diff
@@ -22,20 +29,19 @@ namespace Confus
         class ClientConnection
         {
 		private:
-			/// <summary> The type of packet </summary>
-			enum class EPacketType : unsigned char
-			{
-				Message = 1 + ID_USER_PACKET_ENUM
-			};
-
-            /// <summary> The RakNet interface for interacting with RakNet </summary>
-			RakNet::RakPeerInterface* m_Interface = RakNet::RakPeerInterface::GetInstance();
+            RakNet::RakPeerInterface* m_Interface;
 			/// <summary> The messages it was not able to send yet due to not having a connection established </summary>
-			std::queue<std::string> m_StalledMessages;
+			std::queue<RakNet::BitStream> m_StalledMessages;
 			/// <summary> Whether we are connected to a server</summary>
 			bool m_Connected = false;
 
         public:
+            /// <summary> The type of packet </summary>
+            enum class EPacketType : unsigned char
+            {
+                Message = 1 + ID_USER_PACKET_ENUM,
+                PlayerAttack = 3 + ID_USER_PACKET_ENUM
+            };
```

Comment:
Since it's now public, you might as well place it outside of the class, as in it not being nested